go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/pokt-network/tendermint
+module github.com/tendermint/tendermint
```

Comment:
This should be `module github.com/pokt-network/tendermint`

Reply:
I can't rename this to pokt-network for some reason without having go mod issues. Perhaps it's fork specific?